### PR TITLE
Move html into templates

### PR DESCRIPTION
**Pull request by thijskh**
Robustify the templating by moving HTML from translatable strings into the Twig templates. This also allows us to reduce the amount of `|raw` Twig filtering we use, the vast majority of templates do not use `|raw` anymore now.

This may not be fully backwards compatible: those with language string overrides for the affected strings, or with templates based on the changed strings, may need to make updates.

### CODE REVIEW

File: theme/base/templates/modules/Authentication/View/IdentityProvider/perform-request-access.html.twig [hunk 1]
```diff
@@ -3,10 +3,10 @@
 {% block content %}
 <div class="mod-form">
   <h1>{{ 'send_confirm'|trans }}</h1>
-    {{ 'send_confirm_desc'|trans|raw }}
+    {{ 'send_confirm_desc'|trans }}
```
Comment (MKodde) [inline theme/base/templates/modules/Authentication/View/IdentityProvider/perform-request-access.html.twig:6]: Translation dropped the `<p>`aragraph wraparound. Maybe add it back in template?

File: theme/base/templates/modules/Authentication/View/Feedback/authn-context-class-ref-blacklisted.html.twig [hunk 2]
```diff
@@ -7,8 +7,8 @@
 
 {% block errorMessage %}
     {% if withIdpName %}
-        {{ 'error_authn_context_class_ref_blacklisted_desc'|trans({'%idpName%': feedbackInfoMap['identityProviderName']})|raw }}
+        {{ 'error_authn_context_class_ref_blacklisted_desc'|trans({'%idpName%': feedbackInfoMap['identityProviderName']}) }}
```
Comment (MKodde) [inline theme/base/templates/modules/Authentication/View/Feedback/authn-context-class-ref-blacklisted.html.twig:10]: Was previously wrapped in `<p>`

File: theme/base/templates/modules/Authentication/View/Feedback/authn-context-class-ref-blacklisted.html.twig [hunk 3]
```diff
@@ -7,8 +7,8 @@
...
     {% else %}
-        {{ 'error_authn_context_class_ref_blacklisted_desc_no_idp_name'|trans|raw }}
+        {{ 'error_authn_context_class_ref_blacklisted_desc_no_idp_name'|trans }}
```
Comment (MKodde) [inline theme/base/templates/modules/Authentication/View/Feedback/authn-context-class-ref-blacklisted.html.twig:12]: Was previously wrapped in `<p>`

File: theme/base/templates/modules/Authentication/View/Feedback/authorization-policy-violation.html.twig [hunk 4]
```diff
@@ -32,13 +32,13 @@
 
     {% endif %}
     {% if withIdpName and withSpName %}
-        {{ 'error_authorization_policy_violation_desc'|trans({'%spName%': feedbackInfoMap['serviceProviderName'], '%idpName%': feedbackInfoMap['identityProviderName']})|raw }}
+        {{ 'error_authorization_policy_violation_desc'|trans({'%spName%': feedbackInfoMap['serviceProviderName'], '%idpName%': feedbackInfoMap['identityProviderName']}) }}
```
Comment (MKodde) [inline theme/base/templates/modules/Authentication/View/Feedback/authorization-policy-violation.html.twig:35]: We are losing formatting here; we could still use raw, and combine it with strip_tags as such:

`error_authorization_policy_violation_desc| trans | striptags ('<i><br><p') | raw`

If you don't see the relevance; We did add the additional markup for a (UX) reason. Maybe we should inform Arnout and the other more UI/UX focussed PO's about this change? If they don't mind, it's totaly fine with me

File: theme/base/templates/modules/Default/View/Error/error.html.twig [hunk 5]
```diff
@@ -56,7 +56,7 @@
 
             <div class="error-details">
                 <p class="error-details__intro-text">
-                    {{ 'error_feedback_info_intro'|trans|raw }}
+                    {{ 'error_feedback_info_intro'|trans }}
```
Comment (MKodde) [inline theme/base/templates/modules/Default/View/Error/error.html.twig:59]: We are losing markup here. And probably more significantly than most other cases

File: theme/base/templates/modules/Authentication/View/Feedback/missing-required-fields.html.twig [hunk 6]
```diff
@@ -7,12 +7,20 @@
...
-        {{ 'error_missing_required_fields_desc_no_name'|trans|raw }}
+        {{ 'error_missing_required_fields_desc_no_name'|trans }}
     {% endif %}
+
+    <p>
+        <ul>
+            <li>UID</li>
+            <li>schacHomeOrganization</li>
+        </ul>
+    </p>
```
Comment (MKodde) [inline theme/base/templates/modules/Authentication/View/Feedback/missing-required-fields.html.twig:24]: I think the three different conditions should include different attribute examples right?

**Review by MKodde — COMMENTED** [review]
I see why this is an improvement. And removing the `raw` filters should prevent problems in the future with the cases that are not causing any problems right now. 

However we are losing quite some markup this way. If that is acceptable. I'm fine with this too. But we could also think of a way to make the raw filter more safe. For example by providing a safer blacklist (scrapping the script tags and other possible xss avenues).

After seeing that we are losing many `<p>` wraparounds, I stopped 1 by 1 checking each of them. So be warned there.

File: theme/base/templates/modules/Default/View/Error/error.html.twig (hunk 5 above)
Comment (thijskh) [reply theme/base/templates/modules/Default/View/Error/error.html.twig:59]: I've inspected it and there's no discernable visible differences. So the markup was not doing anything and apparently that was acceptable before.

File: theme/base/templates/modules/Authentication/View/Feedback/authn-context-class-ref-blacklisted.html.twig (hunk 2 above)
Comment (thijskh) [reply theme/base/templates/modules/Authentication/View/Feedback/authn-context-class-ref-blacklisted.html.twig:10]: This was not done consistently for the messages. I've gone through all of them and made it consistent now.

File: theme/base/templates/modules/Authentication/View/Feedback/authorization-policy-violation.html.twig (hunk 4 above)
Comment (thijskh) [reply theme/base/templates/modules/Authentication/View/Feedback/authorization-policy-violation.html.twig:35]: It's about one emphasised term. I'm pretty sure it was added for no real reason. I've asked the PM and he agrees it's needless and can be removed.

**Comment by thijskh** [comment]
I've checked the `<p>` and they were applied inconsistently before this PR already. I've now made that consistent.

I've checked and there's no objection to the small loss of markup. Where there was potentially bigger loss I've already solve that in moving the HTML into the templates.

**Pull request merged**